src/hint_cli.py

```diff
@@ -56,9 +60,51 @@ def parse(argv=None):
     return path, config, action, args
 
 
+def path_last_deploy(path):
+    return path + "/.last_deploy"
+
+
+def save_config(path, config, cfg):
```

Comment:
So 'config' is the config name, like 'production', and cfg is its data? 'name' might be clearer than 'config'?

Reply:
thanks Emma - @hillalex had raised a similar concern in a previous PR and so I have renamed this to `config_name` which is more explicit